alacritty_terminal/src/term/mod.rs

```diff
@@ -2089,6 +2102,24 @@ impl<T: EventListener> ansi::Handler for Term<T> {
         trace!("Setting cursor style {:?}", style);
         self.cursor_style = style;
     }
+
+    #[inline]
+    fn push_title(&mut self) {
+        let title = self.title.to_owned();
+        trace!("Pushing '{}' onto title stack", title);
+        if self.title_stack.len() >= TITLE_STACK_MAX_DEPTH {
+            self.title_stack.remove(0);
+        }
+        self.title_stack.push(title);
+    }
+
+    #[inline]
+    fn pop_title(&mut self) {
+        if let Some(popped) = self.title_stack.pop() {
+            trace!("Popping '{}' from title stack", popped);
```

Comment:
I think we should trace whenever this was called, regardless of there actually is a title or not. We can add a separate `trace` to `set_title` maybe if we care about title updates instead of just escape sequences.

Reply:
Agreed. I'll change that.